File: Platron/Tests/CliningBinTest.cs
using System;
using NUnit.Framework;
using NUnit.Allure.Core;
using NUnit.Allure.Attributes;
using Platron.Pages.Helpers;

namespace Platron.Tests
{
    [TestFixture]
    [AllureNUnit]
    [AllureSuite("Platron")]
    public class CliningBinTest : BaseTest
    {
        [Test]
        public void CliningBin()
        {
            Print("CliningBin");
            new DemoshopHelper()
                .ClickInBinOnPage()
                .CheckExistRowsInBin(1)
                .ActionRemoveElementFromBin()
                .CheckExistRowsInBin();
        }
    }
}

File: Platron/src/test/java/CliningBinTest.java
import org.junit.Test;

public class CliningBinTest extends BaseTest{
    @Test
    public void CliningBin() throws Exception {
        System.out.println("CliningBin");
        IHelper er = new DemoshopHelper()
                .ClickInBinOnPage(1)
                .CheckExistRowsInBin(1)
                .ActionRemoveElementFromBin()
                .CheckExistRowsInBin(0);
    }
}
